<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dimensionado de lote</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>341</width>
      <height>291</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selecciona uno de los métodos de dimensionado.
Rellena los campos indicados en la pestaña correspondiente.</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_eoq">
     <attribute name="title">
      <string>EOQ</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QLineEdit" name="line_cost_emi"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_dem_anu"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_producto"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="line_cost_pos"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nombre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Demanda Anual</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Costes emision</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Costes posesión</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_poq">
     <attribute name="title">
      <string>POQ</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Costes emision</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="line_deman_diaria"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_dem_anu_3"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Costes posesión</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="line_cost_emi_3"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="line_cost_pos_3"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Demanda diaria</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_producto_3"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Nombre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Demanda Anual</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="line_prod_diaria"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Producción diaria</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_eoqR">
     <attribute name="title">
      <string>EOQ Ruptura</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Demanda Anual</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Costes emision</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Costes posesión</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="line_cost_emi_4"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_dem_anu_4"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="line_cost_rupt"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="line_cost_pos_4"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Nombre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Coste ruptura</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_producto_4"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_poqR">
     <attribute name="title">
      <string>POQ Ruptura</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>301</width>
        <height>251</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="2" column="0">
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Costes emision</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Costes posesión</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Demanda Anual</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_26">
         <property name="text">
          <string>Producción diaria</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Demanda diaria</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="line_cost_emi_5"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="line_prod_diaria_2"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="line_cost_pos_5"/>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="line_deman_diaria_2"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="line_producto_5"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_dem_anu_5"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Nombre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="line_cost_rupt_2"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_27">
         <property name="text">
          <string>Coste ruptura</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selección del método de dimensionado:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>-10</y>
      <width>21</width>
      <height>401</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_calcular_tam_lote">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Calcular el tamaño de lote según el método seleccionado en la pestaña.</string>
    </property>
    <property name="text">
     <string>Calcular
Tamaño Lote</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_tam_lote">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_planificacion">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Abre la ventana que muestra la planificación y calcula el coste total.
Es necesario calcular el tamaño de lote previamente.</string>
    </property>
    <property name="text">
     <string>Planificación y Coste Total</string>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_nombre">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>270</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_apell">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Apellido...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_DNI">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>DNI...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_mostrar_fact">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>290</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Abre la ventana que muestra los datos de facturación.</string>
    </property>
    <property name="text">
     <string>Mostrar
Factura</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>unidades</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_coste_total">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>190</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>€</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Información del cliente:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
